input/fsh/DEF_CodeSystems.fsh

```diff
@@ -0,0 +1,106 @@
+Alias: CC = http://hl7.org/fhir/us/mcode/CodeSystem/mcode-catch-code-cs
+Alias: RT = http://terminology.hl7.org/CodeSystem/radiotherapy
+
+CodeSystem: CatchCodeCS
+Id: mcode-catch-code-cs
+Title: "mCODE Catch Codes"
+Description: "Codes needed for positive identification of certain types of instances, more specifically, to uniquely associate instances with slices when they are used in bundles. The codes in this code system are used when a concept falls outside of the defined set of codes. They make it possible to use 'required' bindings where 'extensible' bindings would result an instance matching multiple slices and thus trigger a validation error."
+* #PCC-OTHER "Other Primary Cancer Condition, Specify" "Code to be used when the desired concept is not represented base value set. This code represents the equivalent of 'Other, specify: _________'. Specify the additional concept as the second coding."
+* #SCC-OTHER "Other Secondary Cancer Condition, Specify" "Code to be used when the desired concept is not represented base value set. This code represents the equivalent of 'Other, specify: _________'. Specify the additional concept as the second coding."
+* #TMT-OTHER "Other Tumor Marker Test, Specify" "Code to be used when the desired concept is not represented base value set. This code represents the equivalent of 'Other, specify: _________'. Specify the additional concept as the second coding."
+* #CD-OTHER "Other Cancer Disorder, Specify"  "Code to be used when the desired concept is not represented base value set. This code represents the equivalent of 'Other, specify: _________'. Specify the additional concept as the second coding."
+
+CodeSystem: ResourceIdentifierCS
+Id: mcode-resource-identifier-cs
+Title: "mCODE Resource Identifier Code System"
+Description: "Concepts describing types of instances, to be used in the 'code' element of an instance."
+* #mcode-patient "mCODE Patient Group Resource" "Identifies a Group resource containing mCODE cancer patients that conforms to the MCODEPatientGroup profile."
+* #mcode-comorbidity  "Elixhauser Comorbidity Resource" "Identifies an Observation resource representing Elixhauser comorbidities that conforms to the ComorbiditiesElixhauser profile."
+* #mcode-radiotherapy-summary "Radiotherapy Summary Resource" "Identifies an Observation resource that summarizes a radiotherapy course of treatment that conforms to the RadiotherapySummary profile."
+* #mcode-radiotherapy-ebrt "Teleradiotherapy Prescription Delivery Resource" "Identifies an Observation resource that describes delivery of a teleradiotherapy (external beam) prescription and conforms to the TeleradiotherapyPrescriptionDelivery profile."
+* #mcode-radiotherapy-brachy "Brachytherapy Prescription Delivery Resource" "Identifies an Observation resource that describes delivery of a brachytherapy (external beam) prescription and conforms to the BrachytherapyPrescriptionDelivery profile."
```

Comment:
Should these codes be all caps to match the pattern with the other code systems?

Reply:
Maybe the other way around, but it isn't incumbent on a code system to format its codes in any particular way. They vary all over the map.